content/review/document-level-permissions.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Document level Permissions"
+metaTitle: "sensenet - Document level Permissions"
+metaDescription: "sensenet Document level Permissions"
+---
+
+sensenet offers you the ability of setting permissions even on document level in addition to the [role-based permission](/concepts/role-based-permissions) logic.
+# Advantage
+Using content (or document) level permissions, you can set who has access to an individual content and allow/restrict actions one by one.
+
+# Built-in and custom permissions
+
+There is a set of basic built-in permissions, such as See, Open, Save, Delete etc., and it is even possible to add custom ones as well.
+•	See: you only have the information that the content exists (content metadata cannot be accessed)
+•	Open: you have access to content metadata
+•	Save: you can make changes in the content metadata
+•	Other permissions for removing or approving a content or managing lists and workspaces.
```

Comment:
```suggestion
•	Other permissions for removing or approving a content or managing lists and workspaces.

It is possible to add custom permissions as well. Besides the built-in permission types (See, Open, etc.) sensenet defines 14 custom permission types (Custom01, Custom02, etc.) that can be used for custom permission settings. A common example would be to allow a user to print a document (or deny it): for this developers could assign the permission type Custom1 and use it consistently in the system to check permissions and allow or disallow printing.

It is possible to manage content (document) level permission settings through the admin surface, or with User Management and Permission APIs.
```

Reply:
I forgot to mention here the [local only permissions](https://wiki.sensenet.com/Permission_System.html#Local_only_permissions_.28from_version_6.3.29), that could be added here as well in a separate paragraph